src/jenkins/Messages.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package jenkins
```

Comment:
Shall we call this package jenkins.messages or similar in case there are more lib being written later on?

Reply:
I'll defer this to another PR, otherwise I have to go back retest everything after the rename and it's a lot of busy work.